plugins/stackstorm/src/plugin.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createApiFactory,
+  createPlugin,
+  createRoutableExtension,
+  discoveryApiRef,
+  identityApiRef,
+} from '@backstage/core-plugin-api';
+import { stackStormApiRef, StackStormClient } from './api';
+
+import { rootRouteRef } from './routes';
+
+/**
+ * The Backstage plugin that holds stackstorm specific components
+ *
+ * @public
+ */
+export const stackstormPlugin = createPlugin({
+  id: 'stackstorm',
+  apis: [
+    createApiFactory({
+      api: stackStormApiRef,
+      deps: {
+        discoveryApi: discoveryApiRef,
+        identityApi: identityApiRef,
+      },
+      factory: ({ discoveryApi, identityApi }) =>
+        new StackStormClient({
```

Comment:
I see that the product name is `StackStorm`, with two uppercase S. We try to maintain the exact product capitalization in regular human readable text (comments, user interface text etc), but treated as a single word in code identifiers. You have a bit of mixture in code here. I'd expect `stackstormApiRef`, `StackstormClient`, `StackstormHome`, etc. It's a bit of a nit but also convenient to get correct right away! If you don't mind, maybe you could search for the word stackstorm case insensitively in its entire plugin folder here and see that they get consistent? Same thing goes for the casing of the corresponding filenames.

I know this can get cumbersome. If you don't have the time or feel it's excessively nitpicky, feel free to ignore.

Reply:
Yeah wasn't sure what case to use in places. Done. At least plugin name used the right case 🤣